backend/src/nodes/utils/auto_split.py

```diff
@@ -12,6 +12,34 @@ class Split:
     pass
 
 
+def estimate_tile_size(
+    free: int,
+    total: int,
+    model_size: int,
+    img: np.ndarray,
+    img_element_size: int = 4,
+    allowed_usage: float = 0.8,
+) -> Union[int, None]:
```

Comment:
We always return a tile size.

```suggestion
) -> int:
```